routes/index.js

```diff
@@ -19,4 +22,24 @@ router.get('/registration', function(req, res) {
   res.render('registration.njk');
 });
 
+router.post('/badgein',jsonParser,function(req,res){
+  if(!req.body)
+    { 
+     return res.sendStatus(400);
+    }
+  var BadgeNumber = req.body.badgeNumber;
+  if(typeof dbAPI.validateUser === undefined)
+   {
+     var url = '/registration' + req.body.badgeNumber;
```

Comment:
I believe this URL would need to constructed by adding a `/` in between the string and the badge number but since it is inside this if statement I dont understand under what condition you would need to do this.